### DISCUSSION TITLE
Static image shows up in dev but not in emails.

### DISCUSSION THREAD

**Discussion by calbertan** [resend/react-email#1012] [Q&A]
I have my images in static/logo.png and I am displaying it as below: 
```
<Img src="/static/logo.png" alt="logo" width={58} height={41} className="m-auto"/>
```
The images shows up in the react email dev env but sending the email results in the image not showing up.  I tried changing the source to a live url like localhost:3001/logo.png and that displayed the image fine.

I am using Next.js from my understanding the static folder is deprecated and that may play a role.

**Comment by mamlzy** [comment]
when using `ngrok` my image showing only in resend dashboard but not in gmail.

in gmail:
<img width="794" alt="image" src="https://github.com/resend/react-email/assets/63343105/a5245268-029c-4d5d-a324-81d0166f2ead">

in resend dashboard:
<img width="751" alt="image" src="https://github.com/resend/react-email/assets/63343105/986ef8a6-e2c6-4dcd-aa90-cc1b6c06bf3e">

**Reply by mamlzy** [reply]
update: the image only show on production. i have just tested it.
